plugins/MauticTestBundle/Assets/js/index.js

```diff
@@ -0,0 +1 @@
+console.log('Bundle Plugin Test')
```

Comment:
We need to remove this MauticTestBundle from the PR as we don't want this merged into production.